### PR TITLE
support  doc overview-cn.md translation

### PR DESCRIPTION
**Pull request by Devonmusa**
#Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: docs/others/cn/concepts-and-designs/overview-cn.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
-## 总览
-TODO
\ No newline at end of file
+# 概览
+   SkyWalking一个开源可观测性平台，用于收集、分析、聚合和可视化来自服务和云原生基础设施的数据。SkyWalking提供了一种简单的方法，可以让你清晰的查看分布式系统、甚至是跨云环境中。它更像是现代化APM(Application Performance Manager),专为云原生、基于容器和分布式系统而设计的。
+
+## 为什么要使用SkyWalking？
+   SkyWalking提供了在许多不同场景下观察和监控分布式系统的解决方案。首先，与传统方式不同，SkyWalking为服务实例提供自动埋点代理，例如在Java，C#和NodeJS中。与此同时，他正准备为Go,C++提供手动埋点代理SDK。此外，更多语言需要，运行时代码风险可控，云原生基础设施变得更加强大，SkyWalking使用Service Mesher基础网探针收集数据，来理解整个分布式系统。通常它为**服务**、**服务实例**、**端点**提供可观察性能力。
```
Comment (wu-sheng) [inline docs/others/cn/concepts-and-designs/overview-cn.md]: Serive mesh, 没有er

File: docs/others/cn/concepts-and-designs/overview-cn.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+    
+   `服务`、`服务实例`和`端点`的概念在今天到处使用，所以我们首先要清楚它们在SkyWalking中的含义。
+             
+- **服务**  表示提供相同行为请求的一个`服务集合`或`服务组`工作负载。当你用埋点代理或SDKs时，你可以定义服务名称。或是SkyWalking使用你定义的名字在Istiole这类的平台中。
```
Comment (wu-sheng) [inline docs/others/cn/concepts-and-designs/overview-cn.md]: Istio. 没有le

File: docs/others/cn/concepts-and-designs/overview-cn.md [hunk 3]
```diff
@@ -1,2 +1,30 @@
...
+- **服务实例**  表示`服务组`中的每个工作负载都称作`服务实例`。就像`pods`在Kubernetes中一样，它不需要是OS中的单个进程。然而，如果您正在使用埋点代理，一个服务实例则实际上是OS中的单个进程。
```
Comment (wu-sheng) [inline docs/others/cn/concepts-and-designs/overview-cn.md]: 埋点代理应该翻译成，语言探针

File: docs/others/cn/concepts-and-designs/overview-cn.md [hunk 4]
```diff
@@ -1,2 +1,30 @@
...
+- **端点**  表示某个服务传入请求的路径，例如HTTP URI路径或gRPC服务类+方法签名。
+ 
+通过使用SkyWalking，您可以观察`服务`和`端点`之间的拓扑关系和每个`服务实例`的metrics，并为`服务实例`和`端点`设置警报规则。
+   
+除此之外，您还可以通过使用Zipkin，Jaeger和OpenCensus从SkyWalking本机代理和SDK进行分布式全链路跟踪。
```
Comment (wu-sheng) [inline docs/others/cn/concepts-and-designs/overview-cn.md]: 使用Zipkin...以及SkyWalking的语言探针或者SDK进行分布式全链路追踪

File: docs/others/cn/concepts-and-designs/overview-cn.md [hunk 5]
```diff
@@ -1,2 +1,30 @@
...
+## 架构
+
+   SkyWalking在逻辑上分为四个部分：探针，平台后端，存储和UI。
+   
+   <img src="https://skywalkingtest.github.io/page-resources/6_overview.png"/>
+
+- **探针**  基于不同的来源是不同的。它们收集数据并根据SkyWalking需求重新格式化。
+- **平台后端**  是一个后端服务，支持集群部署。它用于聚合，分析和驱动从探针到UI的流程。它还提供可插拔能力为不同的数据源（如Zipkin）、存储器实现和集群管理。您甚至可以使用[观察分析语言](oal-cn.md)自定义聚合和分析。
+- **存储**  是开放的。您可以选择现有的实现，例如由Sharding-Sphere管理的ElasticSearch、H2或MySQL集群，也可以实现自己的。当然，非常感谢您为新的存储实现做出贡献。
+- **UI**  酷炫且非常强大，你也可以根据自己需要自己定制后端。
```
Comment (wu-sheng) [inline docs/others/cn/concepts-and-designs/overview-cn.md]: 你也可以自己定制UI匹配后端

File: docs/others/cn/concepts-and-designs/overview-cn.md [hunk 6]
```diff
@@ -1,2 +1,30 @@
...
+   
+## 架构
+
+   SkyWalking在逻辑上分为四个部分：探针，平台后端，存储和UI。
+   
+   <img src="https://skywalkingtest.github.io/page-resources/6_overview.png"/>
+
+- **探针**  基于不同的来源是不同的。它们收集数据并根据SkyWalking需求重新格式化。
+- **平台后端**  是一个后端服务，支持集群部署。它用于聚合，分析和驱动从探针到UI的流程。它还提供可插拔能力为不同的数据源（如Zipkin）、存储器实现和集群管理。您甚至可以使用[观察分析语言](oal-cn.md)自定义聚合和分析。
+- **存储**  是开放的。您可以选择现有的实现，例如由Sharding-Sphere管理的ElasticSearch、H2或MySQL集群，也可以实现自己的。当然，非常感谢您为新的存储实现做出贡献。
```
Comment (wu-sheng) [inline docs/others/cn/concepts-and-designs/overview-cn.md]: Shardingsphere只能管理MySQL集群。

Comment (Devonmusa) [reply docs/others/cn/concepts-and-designs/overview-cn.md]: sorry，I will  fix  these problems

File: docs/others/cn/concepts-and-designs/overview-cn.md (hunk 1 above)
Comment (Devonmusa) [reply docs/others/cn/concepts-and-designs/overview-cn.md]: finished it

**Comment by Devonmusa** [comment]
> Review finished. Comments inline.
finished it, I'm sorry，there have so many midtakes，I need to provide quality of work

**Comment by Devonmusa** [comment]
> Review finished. Comments inline.

finished it

**Comment by wu-sheng** [comment]
> finished it, I'm sorry，there have so many midtakes，I need to provide quality of work
 
Don't worry. You just do your best. Improve by multiple reviewers. That is why we always asked for review.

**Pull request merged**